sdk/python/jobs/pipelines/2b_train_cifar_10_with_pytorch/train-model/main.py

```diff
@@ -5,18 +5,22 @@
 # ==============================================================================
 
 # imports
-import os
-import mlflow
 import argparse
+import copy
+import os
 
+import mlflow
+import numpy as np
 import torch
 import torchvision
 import torchvision.transforms as transforms
 import torch.nn as nn
 import torch.nn.functional as F
 import torch.optim as optim
 
-# TODO - add mlflow logging
+
+#- add mlflow logging
```

Comment:
I'm thinking that the difficulty with hard-coding `pipeline_samples` here is that someone could change name of experiment in notebook, but not know to change it in this script. Also, looking at other examples where they log mlflow metrics in this repo, they don't require mlflow.set_experiment? I'm wondering if logging would still work without this...